### ISSUE TITLE
Legend container aria-label not read out in Narrator's Scan Mode

### ISSUE THREAD

**Issue by shandysulen** [microsoft/fluentui#15937]
### Environment Information

- **Package version(s)**: @fluentui/react-charting v5.0.0-beta.0
- **Browser and OS versions**: Microsoft Edge v86.0.622.68, Windows 10 Enterprise 20H2
- **Assistive Technologies**: Narrator

### Describe the issue:

This bug has been tested with the `VerticalStackedBarChart` although I imagine it affects all components that derive from the `CartesianChart` common component.

When Narrator is in Scan Mode (<kbd>Narrator + Space</kbd> to toggle) and reading the whole document (<kbd>Narrator + ↓</kbd>), the `aria-label` passed to the legend container is not read. Without the `aria-label` read out, it can be confusing to understand the purpose of the legend buttons without visual context. The `aria-label` (or `aria-labelledby`) around the legend container is crucial for blind screen reader users, so we should ensure that it's read out in both Scan and Focus mode.

Note that the `aria-label` is read in Focus Mode (<kbd>Narrator + Space</kbd> to toggle).
Note that the screen reader doesn't need to be reading the whole document (<kbd>Narrator + ↓</kbd>) to hear experience this behavior.

### Please provide a reproduction of the issue:

1. Go to first example at [the demo site](https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/charting/dist/index.html#/examples/VerticalStackedBarChart) on the demo site is a perfect showcase.
2. Turn on Narrator (Ctrl + Win + Enter) and enable Read All Document (Narrator + ↓)
3. Listen to the lack of the legend container's `aria-label` being read out

#### Actual behavior:

The `aria-label` of the legend container is **not** read out by the screen reader in Scan mode.

#### Expected behavior:

The `aria-label` of the legend container is read out by the screen reader in Scan mode.

**Comment by paulgildea** [comment]
@shandysulen Thanks for filing this issue!

**Comment by mbest** [comment]
I've talked with a number of people on my team and we don't know of a way to make sure this is read in scan mode.
